src/main/java/org/mskcc/cbio/oncokb/service/SlackService.java

```diff
@@ -261,9 +261,11 @@ public List<LayoutBlock> buildBlocks(UserDTO userDTO, boolean trialAccountActiva
         return blocks;
     }
 
-    private LayoutBlock buildCollapsedBlock(UserDTO userDTO, ActionId actionId) {
+    private LayoutBlock buildCollapsedBlock(UserDTO userDTO, boolean trialAccountActivated, ActionId actionId) {
         StringBuilder sb = new StringBuilder();
-        sb.append(userDTO.getEmail() + "\n" + userDTO.getCompanyName() + " (" + userDTO.getLicenseType().getShortName() + (withNote(DropdownEmailOption.GIVE_TRIAL_ACCESS, userDTO, actionId) ? ", *TRIAL*)" : userDTO.isActivated() ? ")" : ")\n*NOT ACTIVATED*: "));
+        sb.append(userDTO.getEmail() + "\n" + userDTO.getCompanyName() + " (" + userDTO.getLicenseType().getShortName()
+            + ((withNote(DropdownEmailOption.GIVE_TRIAL_ACCESS, userDTO, actionId) && !(userDTO.isActivated() && !trialAccountActivated)) ? ", *TRIAL*)" :
```

Comment:
this `(withNote(DropdownEmailOption.GIVE_TRIAL_ACCESS, userDTO, actionId) && !(userDTO.isActivated() && !trialAccountActivated))` is basically as follows: withNote is if we ever gave them trial access, and second boolean is the negation of conversion to regular account (not converted to regular yet). so the trial note will dissapear if we convert to regular.

Reply:
this logic is used a few other places too